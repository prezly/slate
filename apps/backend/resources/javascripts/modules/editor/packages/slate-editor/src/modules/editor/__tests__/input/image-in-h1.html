<meta http-equiv="content-type" content="text/html; charset=utf-8"><h1 style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-style: initial; text-decoration-color: initial;">lorem<span> </span><img src="https://cdn.uc.assets.prezly.com/631ddf19-df7a-4246-a16f-71c204b3ba66/-/preview/1560x2000/landscape-870x260.jpg"><span> </span>ipsum</h1>
