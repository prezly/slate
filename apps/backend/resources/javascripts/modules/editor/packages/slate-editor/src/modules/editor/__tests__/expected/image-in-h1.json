[
    {
        "type": "paragraph",
        "children": [
            {
                "text": "lorem "
            }
        ]
    },
    {
        "type": "placeholder:image",
        "children": [
            {
                "text": ""
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": " ipsum"
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": ""
            }
        ]
    }
]
